dist/Client.js

```diff
@@ -88,20 +89,24 @@ class APIClient {
       fetch = require('node-fetch'); // https://www.npmjs.com/package/node-fetch
 
       Headers = fetch.Headers;
+      console.debug('Node environment');
     } else {
       fetch = window.fetch; // https://caniuse.com/#feat=fetch, https://github.com/github/fetch
 
       Headers = window.Headers; // https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+
```

Comment:
I think in general in JS, you shouldn't call `console.info/console.debug` from library code as this isn't using some proper logging (like logging module in python) and the application using the lib has no way to control the verbosity.

So unless this is really necessary, I'd just chop all the `console` calls here.

I'm not sure what's the best practice for logging in JS, but I don't want to add a logging library as a dependency, so for now it seems the easiest would be to not do logs.

Reply:
Checked